trackhubs/utils.py

```diff
@@ -0,0 +1,28 @@
+import ast
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def str2obj(var):
+    """
+    Convert string dict/array to an actual dict/array
+    this function is used to convert trackdb_configuration
+    and trackdb_data from string (as stored in MySQL) to JSON
+    (as it need for it to be stored in Elasticsearch when enriching the document)
+    :returns: JSON object
+    """
+    if isinstance(var, dict) or isinstance(var, list):
+        return var
+
+    try:
+        var = ast.literal_eval(var)
+    # if the result of ast.literal_eval is not a dict or array
+    # print the error and return None
+    except ValueError as ex:
+        logger.error(ex)
+        return None
```

Comment:
There are inputs to `ast.literal_eval` which can produce outputs that are not lists/dicts of strings, for example `ast.literal_eval("'a'")` evaluates to 'a', and `ast.literal_eval("1")` evaluates to 1.  I was thinking of adding another if statement underneath line 25 like this:
```
if not (isinstance(var, list) or isinstance(var, dict)):
    return None
```